ccpp/data/GFS_typedefs.meta

```diff
@@ -1860,6 +1860,78 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[emdust]
+  standard_name = emission_of_dust_for_smoke
+  long_name = emission of dust for smoke
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (do_smoke_coupling)
+[emseas]
+  standard_name = emission_of_seas_for_smoke
+  long_name = emission of seas for smoke
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (do_smoke_coupling)
+[emanoc]
+  standard_name = emission_of_anoc_for_thompson_mp
```

Comment:
Same comment RE: `anoc`. Also, why is this specific to Thompson MP? Could another MP scheme conceivably use this? If so, remove `for_thompson_mp`.

Reply:
This is anthropogenic aerosol emission for indirect feedback. It could be used by other MPs. Just because we applied it in Thompson MP. I will update this.